dali/base/dasess.cpp

```diff
@@ -525,11 +519,18 @@ class CSessionRequestServer: public Thread
                 int role=0;
                 if (mb.length()-mb.getPos()>=sizeof(role)) { // a capability block present
                     mb.read(role);
-                    if (!manager.authorizeConnection(role,false)) {
+                    if (!manager.authorizeConnection(node, (DaliClientRole) role))
```

Comment:
In 7.2 and earlier this always returns true, so the changes below will not have any effect. I left them in so that anyone looking at the 7.2 code sees why the deserialize is there.